### PR TITLE
Add note to manual re default memory units of KiB

### PR DESCRIPTION
**Pull request by sparrowt**
Addresses https://github.com/htop-dev/htop/issues/1340

Hopefully this will help newcomers.

### CODE REVIEW

**Review by BenBE — APPROVED** [review]

File: htop.1.in [hunk 1]
```diff
@@ -656,10 +656,12 @@ Memory sizes in
 .B htop
 are displayed in a human-readable form.
 Sizes are printed in powers of 1024. (e.g., 1023M = 1072693248 Bytes)
+If no suffix is shown the units are implicitly KiB.
 .LP
 The decision to use this convention was made in order to conserve screen
 space and make memory size representations consistent throughout
-.BR htop .
+.BR htop
```
Comment (fasterit) [inline htop.1.in:663]: .B not .BR

File: htop.1.in [hunk 2]
```diff
@@ -656,10 +656,12 @@ Memory sizes in
 .B htop
 are displayed in a human-readable form.
 Sizes are printed in powers of 1024. (e.g., 1023M = 1072693248 Bytes)
+If no suffix is shown the units are implicitly KiB.
```
Comment (fasterit) [inline htop.1.in:659]:
```suggestion
If no suffix is shown the units are implicitly K as in KiB (Kibibyte, 1 KiB = 1024 Bytes).
```

**Review by fasterit — CHANGES_REQUESTED** [review]
please see inline comments

File: htop.1.in (hunk 2 above)
Comment (Explorer09) [reply htop.1.in:659]: @fasterit "bytes", when spelled in full, should be all lowercase, not title case.
Same goes for "kibibytes".

File: htop.1.in [hunk 3]
```diff
@@ -656,10 +656,12 @@ Memory sizes in
...
 .LP
 The decision to use this convention was made in order to conserve screen
 space and make memory size representations consistent throughout
-.BR htop .
+.BR htop
+as allocations are granular to full memory pages, typically 4KiB.
```
Comment (Explorer09) [inline htop.1.in:664]:
```suggestion
as allocations are granular to full memory pages (4 KiB for most platforms).
```
The idea is to hint that the page sizes depends on platform.

File: htop.1.in [hunk 4]
```diff
@@ -656,10 +656,12 @@ Memory sizes in
 .B htop
 are displayed in a human-readable form.
 Sizes are printed in powers of 1024. (e.g., 1023M = 1072693248 Bytes)
```
Comment (Explorer09) [inline htop.1.in:658]: If we are mentioning the default unit is "K as in KiB", then the "e.g." can be removed for brevity (as 1 KiB = 1024 bytes is noted right below)
```suggestion
Sizes are printed in powers of 1024.
```

Comment (BenBE) [reply htop.1.in:658]: That at least resolves the long-standing nitpick, that the period should go after the parens … ;-) Anyway, what about mentioning IEC units?

**Comment by sparrowt** [comment]
I have hopefully addressed all the feedback now

**Review by Explorer09 — APPROVED** [review]

**Pull request merged**